doc/source/cluster/kubernetes/user-guides/pod-command.md

```diff
@@ -1,18 +1,69 @@
 (kuberay-pod-command)=
 
 # Specify container commands for Ray head/worker Pods
-You can execute commands on the head/worker pods at two timings:
 
-* (1) **Before `ray start`**: As an example, you can set up some environment variables that will be used by `ray start`.
+KubeRay generates a `ray start` command for each Ray Pod.
+Sometimes, you may want to execute certain commands either before or after the ray start command, or you may wish to define the container's command yourself.
+This document shows you how to do that.
 
-* (2) **After `ray start` (RayCluster is ready)**: As an example, you can launch a Ray serve deployment when the RayCluster is ready.
+## Part 1: Specify your custom container command without any generated `ray start` command
 
-## Current KubeRay operator behavior for container commands
-* The current behavior for container commands is not finalized, and **may be updated in the future**.
-* See [code](https://github.com/ray-project/kuberay/blob/47148921c7d14813aea26a7974abda7cf22bbc52/ray-operator/controllers/ray/common/pod.go#L301-L326) for more details.
+Starting with KubeRay v1.1.0, if users add the annotation `ray.io/overwrite-container-cmd: "true"` to a RayCluster, KubeRay respects the container `command` and `args` as provided by the users, without including any generated command.
```

Comment:
```suggestion
Starting with KubeRay v1.1.0, if users add the annotation `ray.io/overwrite-container-cmd: "true"` to a RayCluster, KubeRay respects the container `command` and `args` as provided by the users, without including the generated `ray start` command.
```
Nit. Just to be a bit more explicit.

Reply:
It would also be helpful to include the definition/explanation of `KUBERAY_GEN_RAY_START_CMD` here, in the text, instead of just in the code comment (users might not expect to see a key definition in the code comment instead of the main text)

Reply:
This is not precise enough. The generated commands include `ulimit` and `ray start` commands. I add more details about both of them.